SimPEG/EM/TDEM/ProblemTDEM.py

```diff
@@ -21,6 +21,7 @@ class BaseTDEMProblem(Problem.BaseTimeProblem, BaseEMProblem):
     surveyPair = SurveyTDEM  #: A SimPEG.EM.TDEM.SurveyTDEM Class
     fieldsPair = FieldsTDEM  #: A SimPEG.EM.TDEM.FieldsTDEM Class
     clean_on_model_update = ['_Adcinv']  #: clear DC matrix factors on any model updates
+    dt_threshold = 1e-8
```

Comment:
@lheagy this prevents doing unnecessary factorization. Let me know what do you think